### PR TITLE
updated cluster doc and examples  for the new IOPS change

### PR DESCRIPTION
**Pull request by nikhil-mongo**
## Description

updated cluster doc for the new IOPS change and removed the IOPS config from examples so that they are launched with the default configuration of 3000 IOPS

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Documentation fix/enhancement

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

File: website/docs/r/cluster.html.markdown [hunk 1]
```diff
@@ -298,6 +298,7 @@ But in order to explicitly change `provider_instance_size_name` comment the `lif
     - AZURE - Microsoft Azure
 
 * `provider_disk_iops` - (Optional - AWS Only) The maximum input/output operations per second (IOPS) the system can perform. The possible values depend on the selected `provider_instance_size_name` and `disk_size_gb`.  This setting requires that `provider_instance_size_name` to be M30 or greater and cannot be used with clusters with local NVMe SSDs.  The default value for `provider_disk_iops` is the same as the cluster tier's Standard IOPS value, as viewable in the Atlas console.  It is used in cases where a higher number of IOPS is needed and possible.  If a value is submitted that is lower or equal to the default IOPS value for the cluster tier Atlas ignores the requested value and uses the default.  More details available under the providerSettings.diskIOPS parameter: [MongoDB API Clusters](https://docs.atlas.mongodb.com/reference/api/clusters-create-one/)
+  * The default IOPS value for the AWS clusters is now 3000 upto 1TB standard EBS storage. You do not need to configure IOPS for the Standard EBS configuration.
```
Comment (themantissa) [inline website/docs/r/cluster.html.markdown]: We should not add all of this - as noted above they should view in the Atlas console as we can't keep up with changes in a human generated fashion in docs right now.   
If you want to add "You do not need to configure IOPS for a STANDARD disk configuration but only for a PROVISIONED configuration. " I think that would make sense.

File: examples/starter/atlas_cluster.tf [hunk 2]
```diff
@@ -17,7 +17,6 @@ resource "mongodbatlas_cluster" "cluster" {
   auto_scaling_disk_gb_enabled = true
   provider_name                = "AWS"
   disk_size_gb                 = 10
-  provider_disk_iops           = 100
   provider_volume_type         = "STANDARD"
```
Comment (themantissa) [inline examples/starter/atlas_cluster.tf]: remove provider_volume_type as it's not needed

File: examples/aws-atlas-privatelink/atlas-cluster.tf [hunk 3]
```diff
@@ -17,7 +17,6 @@ resource "mongodbatlas_cluster" "cluster-atlas" {
   //Provider settings
   provider_name               = "AWS"
   disk_size_gb                = 10
-  provider_disk_iops          = 100
   provider_volume_type        = "STANDARD"
```
Comment (themantissa) [inline examples/aws-atlas-privatelink/atlas-cluster.tf]: remove as not needed: provider_volume_type

File: examples/Atlas_Database_Users/atlas_cluster.tf [hunk 4]
```diff
@@ -17,7 +17,6 @@ resource "mongodbatlas_cluster" "cluster" {
   auto_scaling_disk_gb_enabled = true
   provider_name                = "AWS"
   disk_size_gb                 = 10
-  provider_disk_iops           = 100
   provider_volume_type         = "STANDARD"
```
Comment (themantissa) [inline examples/Atlas_Database_Users/atlas_cluster.tf]: Remove as not needed.

**Review by themantissa — CHANGES_REQUESTED** [review]
A few comments in line, thank you for doing this.

**Comment by nikhil-mongo** [comment]
@themantissa the changes have been made.

**Review by themantissa — APPROVED** [review]
Great job, thank you @nikhil-mongo !

**Pull request merged**